ops.yml

```diff
@@ -1,27 +1,27 @@
-version: "2"
+version: '2'
 pipelines:
-  - name: build-sample-expressjs-app:0.4.4
-    description: build sample expressjs application
+  - name: sample-app-aws-ecs-fargate:0.1.2
+    description: build sample application
     env:
       static:
+        - AWS_REGION=us-west-1
         - DOCKER_REGISTRY=ghcr.io
-        - GITHUB_ORG=biagotaski
-        - GITHUB_REPO=sample-app
+        - ORG=workflows-sh
+        - REPO=sample-app-aws-ecs-fargate
+        - STACK_TYPE=aws-ecs-fargate
       secrets:
-      - DOCKER_REGISTRY_USER
-      - DOCKER_REGISTRY_PASS
-      - GH_USERNAME
-      - GH_TOKEN
+        - AWS_ACCESS_KEY_ID
+        - AWS_ACCOUNT_NUMBER
+        - AWS_SECRET_ACCESS_KEY
+        - GITHUB_TOKEN
```

Comment:
I think this should be handled in main but please check your conflicts to confirm.